setup.cfg

```
@@ -23,6 +23,7 @@ files =
     uvicorn/protocols/websockets/auto.py,
     uvicorn/supervisors/__init__.py,
     uvicorn/middleware/debug.py,
+    uvicorn/middleware/wsgi.py,
```

```suggestion
    uvicorn/middleware/wsgi.py,
    tests/middleware/test_wsgi.py,
```
let's add the test also, this will show you some easy to spot errors in the types